.github/workflows/diff.yml

```diff
@@ -21,7 +21,7 @@ jobs:
           fetch-depth: 0
       - uses: actions/setup-go@0c52d547c9bc32b1aa3301fd7a9cb496313a4491 # v5.0.0
         with:
-          go-version: '1.21'
+          go-version: '1.22'
```

Comment:
We can use this option: `go-version-file: 'go.mod'`

Reply:
Implemented